### DISCUSSION TITLE
Hidden Timer after Update

### DISCUSSION THREAD

**Discussion by ToniDorp** [Hypfer/Valetudo#1850] [Q&A / Support] [answered]
Hi,

i have updatet my Valetudo on the Roborock S5 from a 0.XX Version to 2023.05. 
I set a new timer like i used on the old version.

I´m now trying to setup a control via Home Assistant and therefore i deletet my Valetudo Timer. But it still starts the cleaning at 10:30 (8:30 UTC).

I dont have setup any timers in Home Assistant and the Timer page in Valetudo is also empty. 

I found via SSH the apptimer.cfg in /mnt/data/rockrobo/
and i found this line:

Y 1619192950425 1 1 30 10 * * 1,2,3,4,5 app_start {"fan_power":102,"segments":"0","repeat":1,"clean_order_mode":1}

Can this be the timer?

Thanks for your help.

![apptimer cfg](https://github.com/Hypfer/Valetudo/assets/80211234/e989e261-4f5e-48f4-9e05-774d03c484af)
![No Timer](https://github.com/Hypfer/Valetudo/assets/80211234/c8fd2675-9fb6-49a8-8dd9-0ab2071c15f5)

**Comment by Hypfer** [comment, marked as answer]
https://github.com/Hypfer/Valetudo/releases/tag/2021.01.1
> Make sure to disable any Timers you might've configured before upgrading, since there's no way to delete/configure them in this release anymore!

You can get rid of the old firmware timers either by factory resetting or by using https://github.com/Hypfer/valetudo-helper-miio
Miio command `get_timer` will show you all timers + IDs which you can then use with the `del_timer` command

See also:
https://github.com/Hypfer/Valetudo/blob/6db8b76bddfea636303d1169fa7441ac21a5deca/lib/devices/Roborock.js#L363-L416

**Reply by ToniDorp** [reply]
Thanks for the fast reply,

i have missed this line in the release tag.

I will try to delete it with the miio command.

Thank you very much.
